src/components/cards/Item.tsx

```diff
@@ -137,16 +137,12 @@ const Item: React.FC<PropTypes> = ({ data, category, history }) => {
                     ) : null}
                 </h4>
             </button>
-            <Button
-                className="m-2 hover:text-primary transition-colors duration-300 text-black absolute right-0"
+            <div
+                className="hover:text-primary transition-colors duration-300 text-black px-0"
                 onClick={addItemToShopList}
-                style={{
-                    height: 'fit-content',
-                    width: 'fit-content',
-                }}
             >
-                <MdAdd size={24} />
-            </Button>
+                <MdAdd className="text-gray-light" size={24} />
+            </div>
```

Comment:
Why did you change this to a div?

Reply:
Because of the default classes in the button which add extra margin/padding that I couldn't control directly in this component.

Reply:
Can we adjust the button component for that? For example, we remove spacing from default classes and add them to props. So now if the spacing is in props use those classes otherwise we just use some default spacing classes.

Reply:
As this button has not the same design as the "standard" buttons, it is not better to modify only that button?

Reply:
I suppose. But still, change it to button element instead of div.